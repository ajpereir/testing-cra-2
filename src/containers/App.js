import React from 'react';

import { ContextProvider } from '../context';

import ChildComponent from '../components/ChildComponent';

class App extends React.Component {
  render() {
    return (
      <ContextProvider>
        <div className="App">
          dfbdfgh
          <ChildComponent />
        </div>
      </ContextProvider>
    );
  }
}

export default App;
