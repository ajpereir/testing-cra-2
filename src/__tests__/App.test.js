/* eslint-disable */
import React from 'react';
import { shallow } from 'enzyme';

import App from '../containers/App';

describe('<App/>', () => {
  const wrapper = shallow(<App />);

  it('matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('contains classname="App"', () => {
    expect(wrapper.find(<div className="App" />));
  });

  // it('has text "Tests"', () => {
  //   const wrapper = shallow(<App />);
  //   expect(wrapper.text()).to.equal('Tests');
  // });
});
