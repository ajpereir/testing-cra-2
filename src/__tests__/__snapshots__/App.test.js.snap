// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> matches snapshot 1`] = `
<ContextProvider>
  <div
    className="App"
  >
    <ComponentWithContext />
  </div>
</ContextProvider>
`;
